src/drivers/driver.ts

```diff
@@ -207,23 +207,24 @@ export abstract class CMakeDriver implements vscode.Disposable {
    * Get the environment variables that should be set at CMake-build time.
    */
   async getCMakeBuildCommandEnvironment(in_env: proc.EnvironmentVariables): Promise<proc.EnvironmentVariables> {
+    let envs: proc.EnvironmentVariables;
```

Comment:
Should we revert this in case we decide to use Config Preset Environment variables using `getConfigureEnvironment `? Or do we still want the `in_env` to be used?